src/redux/actions/projects/createProject.js

```diff
@@ -34,7 +46,40 @@ const createProject = (
   };
 
   try {
-    await dispatch(saveProject(newProjectUuid, newProject));
+    const url = `/v1/projects/${newProjectUuid}`;
+    try {
```

Comment:
I just realized there is a `try` block in a `try` block. Is this needed? Can we reduice it to one `try` block?